tuftool/Cargo.toml

```diff
@@ -34,7 +34,7 @@ serde_json = "1.0.87"
 simplelog = "0.12"
 snafu = { version = "0.7", features = ["backtraces-impl-backtrace-crate"] }
 tempfile = "3.3.0"
-tokio = "~1.18"  # LTS
+tokio = "~1.24"  # LTS
```

Comment:
1.20 is still the "LTS" release, so based on this comment I am not sure if we want to move to 1.24.1 or not. Would be good if someone who has more background context can confirm.

We should at least move to the current LTS release, which would be `"~1.20"`.

Reply:
I left a comment on a previous dependabot PR, but we should decide if we are still planning on waiting for whatever LTS comes after 1.20.x. We'll have to make a decision before June 2023 when 1.18.x hits EOL. The reason for not jumping to 1.20.x is the EOL doesn't buy us much more time (Sept 2023).

Reply:
> The reason for not jumping to 1.20.x is the EOL doesn't buy us much more time (Sept 2023).

On the other hand though, there are other fixes and improvements between the two that would be good to pick up, even if it doesn't buy much additional time for support.